unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Common/Button_OnPointerDown.cs

```diff
@@ -0,0 +1,15 @@
+﻿using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+// In oder to lock/unlock the cursor in the same frame a button is clicked we need to use OnPointerDown (event not accesible in the regular Button)
```

Comment:
I would put this class comment in this way:

`/// <summary>`
`/// In oder to lock/unlock the cursor in the same frame a button is clicked we need to use OnPointerDown (event not accesible in the regular Button)`
`/// </summary>`

This way, the IDE's intellisense will detect the comment as "something to show" when someone is going to use this class, and it will be more useful.

Reply:
done!